extra-network/wireguard/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=wireguard
+PKGVER=0.0.20170613
+PKGSEC=net
+PKGDEP="libmnl dkms linux+api"
+PKGDES="A fast, modern and secure Layer-3 VPN tunnel."
+
+ABTYPE=plainmake
```

Comment:
You already have `build` file, that contradicts with this line. And according to your current build script, you shouldn't be using this line.